substrate/frame/revive/src/evm/runtime.rs

```diff
@@ -696,6 +696,13 @@ mod test {
 			UncheckedExtrinsicBuilder::call_with(RUNTIME_PALLETS_ADDR).data(remark.encode());
 		let (_, call, _, _, _, _) = builder.check().unwrap();
 
-		assert_eq!(call, remark);
+		match call {
+			RuntimeCall::Contracts(crate::Call::eth_substrate_call {
+				call: inner_call, ..
+			}) => {
+				assert_eq!(*inner_call, remark);
+			},
+			_ => assert!(false, "Call does not match."),
```

Comment:
```suggestion
			_ => panic!("Expected the RuntimeCall::Contracts variant, got: {:?}"),
```

Nit, assuming `call` here implements `Debug`.